controller.go

```diff
@@ -149,6 +151,14 @@ func loadSettings() error {
 		return fmt.Errorf("invalid max number of certificates per ALB: %d. AWS does not allow more than %d", maxCertsPerALB, aws.DefaultMaxCertsPerALB)
 	}
 
+	if quietFlag && debugFlag{
+		log.Error("--quiet and --debug flags are both set. Debug will be used as logging level.")
```

Comment:
Should be `Info` or `Warn` IMO.

Reply:
@mikkeloscar so fallback to default InfoLevel?

Reply:
No, sorry, I just mean the log should be an Info log or a Warning, not `Error`, It's fine to fall back to Debug :)